x-pack/plugins/siem/public/containers/hosts/last_first_seen/index.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import ApolloClient from 'apollo-client';
+import { get } from 'lodash/fp';
+import React, { useEffect, useState } from 'react';
+
+import { GetHostLastFirstSeenQuery } from '../../../graphql/types';
+import { inputsModel } from '../../../store';
+import { QueryTemplateProps } from '../../query_template';
+
+import { HostLastFirstSeenGqlQuery } from './last_first_seen.gql_query';
+
+export { HostsFilter } from '../filter';
```

Comment:
I believe this duplicates the `HostsFilter` export from `x-pack/plugins/siem/public/containers/hosts/index.tsx`